parallize.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
```

Comment:
Probably adding some author / revision information as a top level file comment is a good idea. and any other relevant top level description info if you think its nessicary.

Reply:
Author information, date, revision history, etc. is captured by Git. Top level description is in in the docstring in main()